File: code forces318A/src/code/forces318a/CodeForces318A.java
package code.forces318a;
import java.util.Scanner;
public class CodeForces318A {

    public static void main(String[] args) {
        Scanner input=new Scanner (System.in);
        int[] number;
        number = new int[1000];
        int count=0,i,num,num1;
        num = input.nextInt();
        num1 = input.nextInt();
        int y=0;
        for(i=1;i<=num;i++){
            if(i%2!=0){
                number[y]=i;
                y++;count++;
            }
        }
        for(i=1;i<=num;i++){
            if(i%2==0){
                number[y]=i;
                y++;count++;
            }
        }
        for(y=0;y<count;y++){
            if(y==num1-1)System.out.println(number[y]);
        }
    }   
}

File: code forces 318A.cpp
#include<iostream>
using namespace std;
int main()
{
	long number[100000];
	long  num1,num2,i,count=0,y=0;
	cin>>num1>>num2;
	for(i=1;i<=num1;i++){
		if(i%2!=0){
			number[y]=i;
			y++;
			count++;
		}
	}
	for(i=1;i<=num1;i++){
		if(i%2==0){
			number[y]=i;
			y++;
			count++;
		}
	}
	for(y=0;y<count;y++){
		if(y==num2-1)cout<<number[y]<<endl;
	}
	return 0;
}
